cloudify_agent/resources/script/linux-download.sh.template

```diff
@@ -17,7 +17,8 @@ run_as()
 add_ssl_cert()
 {
     # Create all the directories in the path to the cert file
-    mkdir -p $(dirname {{ ssl_cert_path }})
+    sudo mkdir -p $(dirname {{ ssl_cert_path }})
+    sudo chown -R {{ user }}. $(dirname {{ agent_dir }})
```

Comment:
It probably shouldn't be owned by the user, but should be readable by them- perhaps we should add a cloudify-agent group and add the current user to it?
Otherwise, aside from any best practices, we'd have an issue if two different users had agents deployed on the same server.

Reply:
Agree we should allow any user execute and install agent to have access to `/opt/cloudify-agent-{version}` I missed that point will check it